recipes/tox/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "tox" %}
+{% set version = "2.7.0" %}
+{% set sha256 = "9c3bdc06fe411d24015e8bbbab9d03dc5243a970154496aac13f9283682435f9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+    - pip
+    - wheel 
+    - virtualenv >=1.11.2
+    - pluggy >=0.3.0,<1.0
```

Comment:
Please drop everything from `requirements/build` that are not need at build time.